torchani/cuaev/aev.cu

```diff
@@ -759,7 +766,8 @@ Result cuaev_forward(const Tensor& coordinates_t, const Tensor& species_t, const
         aev_t, Tensor(), Tensor(), Tensor(), 0, 0, 0, Tensor(), Tensor(), Tensor(), 0, 0, 0, coordinates_t, species_t};
   }
 
-  cudaStream_t stream = at::cuda::getCurrentCUDAStream();
+  at::cuda::CUDAStream stream = at::cuda::getDefaultCUDAStream(coordinates_t.device().index());
```

Comment:
Why using the default stream? CUDA extension should use whatever stream specified by the user. If the `getCurrentCUDAStream()` does not return the stream on the correct device, you should use `DeviceGuard` on your implementation: https://github.com/pytorch/pytorch/blob/master/c10/core/DeviceGuard.h

Reply:
Fixed